app/views/assessor_interface/preliminary_checks/edit.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :page_title, "#{"Error: " if @form.errors.any?}Preliminary check" %>
+<% content_for :back_link_url, assessor_interface_application_form_path(@form.application_form) %>
+
+<%= form_with model: @form, url: [:assessor_interface, @form.application_form, @form.assessment, :preliminary_check], method: :put do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <h1 class="govuk-heading-xl">Preliminary check</h1>
+  <p class="govuk-body">Applications from this country require a preliminary check of the information that the applicant has provided.</p>
+  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+  <h2 class="govuk-heading-s">Country-specific information shown to applicant</h2>
+  <p class="govuk-body">[TRCN will charge you a fee of ₦60,000 for providing the [name of document].</p>
+  <p class="govuk-body">TCRN will also need to see your QTS application reference number as proof that you’ve applied, as well as copies of your teaching documents.]</p>
+  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+  <p class="govuk-body">Once you’ve carried out this check, you should either recommended that the application moves to a full assessment, or decline it.</p>
+  <p class="govuk-body">If you choose to decline it, you should do this in the appropriate section of ‘Initial assessment’ by selecting a decline reason.</p>
+  <div class="govuk-inset-text">
+    For example, qualifications in a subject we cannot accept would mean selecting a decline reason in the ‘Check qualifications’ section.
+  </div>
+
+  <%= f.govuk_radio_buttons_fieldset(
+        :preliminary_check_complete,
+        legend: {
+          text: "Do you want to move this application to a full assessment?",
+          size: "s"
+        }) do %>
+    <%= f.govuk_radio_button(
+          :preliminary_check_complete,
+          :true,
+          link_errors: true,
+          label: { text: "Yes" },
+          ) %>
+    <%= f.govuk_radio_button(
+          :preliminary_check_complete,
+          :false,
+          label: { text: "No, I’ll decline it in the appropriate section" },
+          ) %>
+  <% end %>
```

Comment:
```rb
  <%= f.govuk_collection_radio_buttons :preliminary_check_complete,
                                       [OpenStruct.new(value: :true), OpenStruct.new(value: :false)],
                                       :value,
                                       legend: { size: "s" } %>
```

An alternative way of rendering the radio button would be this, combined with putting the strings in the locale file (for example [the exemption options](https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/cf7bed35f5e803945945b2aa8be8bbafb5e0d012/config/locales/helpers.en.yml#L134-L137)) which keeps the view specific to the structure (and in theory makes it easier for a content designer to make changes to the content via the locale file).

I don't mind too strongly, just a suggestion as I've been trying to move us towards using the locale files more.